.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,78 @@
+---
+name: "Bug Report"
+description: "Report a bug to help us improve."
+title: "[Bug]: <title>"
+type: "Bug"
+body:
+  - type: markdown
+    attributes:
+      value: "Please fill out the following details to help us address the issue."
+  - type: input
+    id: title
+    attributes:
+      label: "Bug Title"
+      description: "Provide a short and descriptive title for the bug."
+      placeholder: "e.g., App crashes on startup"
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: "Bug Description"
+      description: "Describe the bug in detail. Include steps to reproduce, expected behavior, and actual behavior."
```

We have steps to reproduce section further down, so we shouldn't ask for that here.

```suggestion
      description: "Describe the bug in detail. Include the expected behavior and actual behavior."
```